args.go

```diff
@@ -9,6 +9,11 @@ import (
 	"github.com/valyala/bytebufferpool"
 )
 
+const (
+	ArgsNoValue = true
```

Comment:
why may we need to introduce such variables in the public API?

Reply:
Good point, I think they should be made internal only.